chapters.md

```diff
@@ -189,6 +189,28 @@ Chapter 2                | false              | no
 ++Nested Chapter 2.2.1   | true               | no
 ++Nested Chapter 2.2.2   | false              | no
 
+### ChapterSegmentUID
+
+
+The `ChapterSegmentUID` is a binary value with a size of 128 bits and the base element to set up a `Linked Chapter` in 2 variations. For both variations the following 3 conditions MUST be met.
+
+ 1. The `EditionFlagOrdered Flag` is set to true
+ 2. The `ChapterSegmentUID` is not the own `SegmentUID`
+ 3. The linked Matroska file/Segment is in the same folder
+  
+#### Variation 1: Linked-Duration
+
+Two more conditions MUST be met.
+
+ 1. `ChapterTimeStart` and `ChapterTimeEnd` timestamps MUST be in the range of the linked Matroska file/Segment duration
```

Comment:
Do not use "file" here this is adding more confusion than it helps. We usually try to avoid it. Especially as linked segments often come concatenated in a single file.